tools/ci.sh

```diff
@@ -378,20 +346,20 @@ function examples_valgrind {
     mkdir -p build; cd build; rm -rf *
 
     # create certificates for the examples
-    python3 ../tools/certs/create_self-signed.py -c server 
-    python3 ../tools/certs/create_self-signed.py -c client 
+    python3 ../tools/certs/create_self-signed.py -c server
+    python3 ../tools/certs/create_self-signed.py -c client
+
+    # copy json server config
+    cp ../examples/file_based_server/server_config.json5 server_config.json5
```

Comment:
Don't hide the default config in the examples.
Put it in the plugins folder next to the parse code.